kolibri/core/logger/api.py

```diff
@@ -871,6 +883,51 @@ class MasteryLogViewSet(ReadOnlyValuesViewset):
         "mastery_level",
         "complete",
     )
+    summary_values = (
+        "id",
+        "start_timestamp",
+        "completion_timestamp",
+    )
+
+    def __init__(self, *args, **kwargs):
+        super(MasteryLogViewSet, self).__init__(*args, **kwargs)
+        self.attempt_log_view_set = AttemptLogViewSet()
+
+    @action(detail=False)
+    def summary(self, request):
+        user_id = request.GET.get("user")
+        content_id = request.GET.get("content")
+
+        # required parameters
+        if not user_id:
+            raise Http404("Parameter `user` is required")
```

Comment:
Not blocking - I've used 412 previously for missing parameters (e.g. in the required parameters decorator https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/decorators.py#L35 - which I realize I have failed to make usable just specific endpoint methods).